src/SFA.DAS.FindEmploymentSchemes.Contentful/GdsHtmlRenderers/GdsEmbeddedYoutubeContentRenderer.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Contentful.Core.Models;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.FindEmploymentSchemes.Contentful.GdsHtmlRenderers
+{
+    /// <summary>
+    /// A renderer for an embedded YouTube video
+    /// </summary>
+    public class GdsEmbeddedYoutubeContentRenderer : IContentRenderer
+    {
+        private readonly ContentRendererCollection _rendererCollection;
+
+        /// <summary>
+        /// Initializes a new GdsEmbeddedYoutubeContentRenderer
+        /// </summary>
+        /// <param name="rendererCollection">The collection of renderer to use for sub-content.</param>
+        public GdsEmbeddedYoutubeContentRenderer(ContentRendererCollection rendererCollection)
+        {
+            _rendererCollection = rendererCollection;
+        }
+
+        /// <summary>
+        /// The order of this renderer in the collection.
+        /// </summary>
+        public int Order { get; set; } = 40;
+
+        /// <summary>
+        /// Whether or not this renderer supports the provided content.
+        /// </summary>
+        /// <param name="content">The content to evaluate.</param>
+        /// <returns>Returns true if the content is a paragraph, contains only an iframe and refers to a youtube embedded url, otherwise false.</returns>
+        public bool SupportsContent(IContent content)
+        {
+            if (!(content is Paragraph))
+                return false;
+
+            Paragraph paragraph = (Paragraph)content;
+            if (paragraph.Content.Count != 1 || !(paragraph.Content[0] is Text))
+                return false;
+
+            string text = ((Text)paragraph.Content[0]).Value;
+            return text.StartsWith("<iframe") && text.EndsWith("</iframe>") && text.Contains("youtube.com/embed/");
```

Comment:
Missed this on the original PR: we should be using youtube-noccokie.com, so that YouTube doesn't create cookies, and we don't have to add their cookies on our cookies page.
(See [Privacy enhanced mode](https://support.google.com/youtube/answer/171780?hl=en-GB#zippy=%2Cturn-on-privacy-enhanced-mode).)

Reply:
Changed to use youtube-nocookie.com domain